.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname $0)/_/husky.sh"
 
-yarn format && yarn lint
```

Comment:
This is a breaking change. yarn format can now fail, and the pre-commit script still passes. Probably best to add `set -x` to this script.